docs/API.md

```diff
@@ -6,6 +6,10 @@ These are best practices related to the current API used for App. This does not
 - There should be no client-side handling that is unique to any API call.
 - Data is pushed to the client and put straight into Onyx by low-level libraries.
 - Clients should be kept up-to-date with many small incremental changes to data.
+- Creating data needs to be flawlessly fast on every connection (offline, slow 3G, etc).
+- For new objects created from the client (reports, reportActions, policies) we're going to generate a random string ID immediately on the client, rather than needing to wait for the server to give us an ID for the created object.
+- Every new command added should be done right the first time (following the correct pattern and not worrying about unblocking a newsletter initiative quickly).
```

Comment:
https://expensify.slack.com/archives/C02HWMSMZEC/p1655484025907299?thread_ts=1655245003.452099&cid=C02HWMSMZEC

Reply:
This point seems a little pointless to me. I think every PR, no matter what, should be done right the first time. There is no PR that shouldn't be done right the first time. I suggest removing this point.